projects/packages/connection-ui/src/class-admin.php

```diff
@@ -13,18 +13,37 @@
 class Admin {
 
 	/**
-	 * Construction.
+	 * Singleton instance.
+	 *
+	 * @var $instance
 	 */
-	public function __construct() {
-		add_action( 'admin_menu', array( $this, 'register_submenu_page' ), 1000 );
-		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+	private static $instance = null;
+
+	/**
+	 * No instantiation, use `get_instance()`.
+	 */
+	private function __construct() {}
```

Comment:
Can this be removed, or are there plans to add behavior to instantiated classes later?

Reply:
Thanks for reviewing Steve. For singleton pattern you want to block instantiation of the class which is why it's private. But as Sergey mentions below, there is a better solution!